indico/MaKaC/PDFinterface/conference.py

```diff
@@ -443,15 +444,10 @@ class AbstractBook(ContributionBook):
     _tpl_filename = "book_of_abstracts.tpl"
     _table_of_contents = True
 
-    def __init__(self, conf, aw, tz=None):
-        if not tz:
-            tz = conf.getTimezone()
-
-        sort_by = conf.getBOAConfig().getSortBy()
-        if not sort_by.strip() or sort_by not in ["number", "name", "sessionTitle", "speaker", "schedule"]:
-            sort_by = "number"
+    def __init__(self, conf):
```

Comment:
take the `event` here; let's use legacy conference objects as rarely as possible